bowling/tests/bowling_alley_tests_gt.cpp

```diff
@@ -0,0 +1,21 @@
+#include "gtest/gtest.h"
+
+#include "BowlingAlley.hpp"
+
+#include <filesystem>
+
+class BowlingAlleyTests : public ::testing::Test {
+protected:
+    std::filesystem::path path = "../games";
+    BowlingAlley bowlingAlley{path};
+
+};
+
+TEST_F(BowlingAlleyTests, bowlingAlleyCanBeDefaultCreated) {
+    BowlingAlley alley;
+}
+
+TEST_F(BowlingAlleyTests, bowlingAlleyCanBeLoadedFromFolder) {
+    BowlingAlley alley;
+    alley.loadFromFolder(path);
+}
```

Comment:
well... you have a class that is not testable. Have you worked in TDD?
At least printOutput function could return string (or you can have a new function that does it and it is used in printOutput). You would be able to test the output of the whole game.